nevergrad/functions/images/imagelosses.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from koncept.models import Koncept512 as K512_model
+
+
+class ImageLoss:
+    def __init__(self, reference=None) -> None:
+        self.reference = reference
+
+    def __call__(self, img: np.ndarray) -> float:
+        raise NotImplementedError
+
+
+class SumAbsoluteDifferences(ImageLoss):
+    def __init__(self, reference: np.ndarray) -> None:
+        super().__init__(reference)
+        self.domain_shape = self.reference.shape
+
+    def __call__(self, x: np.ndarray) -> float:
+        assert x.shape == self.domain_shape, f"Shape = {x.shape} vs {self.domain_shape}"
+        value = float(np.sum(np.fabs(x - self.reference)))
```

Comment:
I reverted it since np.linalg.norm does max(sum(abs(x), axis=0)), which is different.

Reply:
waouh that's sad, I just read the doc 3 times now and i don't understand why they did this, especially since for numbers different from 1 and 2 they do the expected computation: `sum(abs(x)**ord)**(1./ord)`
Sorry for the mistake